generative_ai/requirements.txt

```
@@ -1,8 +1,8 @@
 pandas==1.3.5; python_version == '3.7'
 pandas==2.0.3; python_version == '3.8'
 pandas==2.1.4; python_version > '3.8'
-pillow==9.5.0; python_version < '3.8'
-pillow==10.3.0; python_version >= '3.8'
+pillow==10.3.0; python_version < '3.8'
```

pillow 10.0.0 dropped support for 3.7; shouldn't bump this
(even though we don't currently test 3.7, it's good to make sure we're not introducing incorrect code)
```suggestion
pillow==9.5.0; python_version < '3.8'
```